src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/edit_index_pattern.js

```diff
@@ -140,4 +142,45 @@ uiModules.get('apps/management')
       $scope.indexPattern.timeFieldName = field.name;
       return $scope.indexPattern.save();
     };
+
+    $scope.tryToRenderScriptedFieldsTable = function () {
```

Comment:
Does this need to be a scope method? Can we just make it a local function?

Reply:
And a minor nit: can we rename it to `updateScriptedFieldsTab`? I wasn't sure how to intuit its role from the word "try" in the name.

Reply:
Sure, I'll change it